src/fides/api/schemas/privacy_experience.py

```diff
@@ -39,73 +66,67 @@ class ExperienceConfigCreate(ExperienceConfigSchema):
 
     @validator("regions")
     @classmethod
     def validate_regions(
         cls, regions: List[PrivacyNoticeRegion]
     ) -> List[PrivacyNoticeRegion]:
         """Assert regions aren't duplicated."""
-        if len(regions) != len(set(regions)):
+        if regions and len(regions) != len(set(regions)):
             raise ValueError("Duplicate regions found.")
         return regions
 
     @root_validator
     def validate_attributes(cls, values: Dict[str, Any]) -> Dict[str, Any]:
-        """Validate minimum set of required fields exist given the type of component and/or delivery_mechanism"""
+        """Validate minimum set of required fields exist given the type of component"""
         component: Optional[ComponentType] = values.get("component")
-        delivery_mechanism: Optional[DeliveryMechanism] = values.get(
-            "delivery_mechanism"
-        )
-
-        if delivery_mechanism == DeliveryMechanism.link and not values.get(
-            "link_label"
-        ):
-            raise ValueError(
-                "Link label required when the delivery mechanism is of type link."
-            )
 
         if component == ComponentType.overlay:
-            if delivery_mechanism == DeliveryMechanism.banner:
-                required_banner_fields = [
-                    "acknowledgement_button_label",
-                    "banner_title",
-                    "confirmation_button_label",
-                    "reject_button_label",
-                ]
-                for field in required_banner_fields:
-                    if not values.get(field):
-                        raise ValueError(
-                            f"The following fields are required when defining a banner: {required_banner_fields}."
-                        )
-
-        if component == ComponentType.privacy_center:
-            if delivery_mechanism != DeliveryMechanism.link:
-                raise ValueError(
-                    "Privacy center experiences can only be delivered via a link."
-                )
+            # Overlays have a few additional required fields beyond the privacy center
+            required_overlay_fields = [
+                "acknowledge_button_label",
+                "banner_enabled",
+                "privacy_preferences_link_label",
+            ]
+            for field in required_overlay_fields:
+                if not values.get(field):
+                    raise ValueError(
+                        f"The following additional fields are required when defining an overlay: {human_friendly_list(required_overlay_fields)}."
+                    )
 
         return values
 
 
 class ExperienceConfigUpdate(ExperienceConfigSchema):
     """
-    Updating ExperienceConfig - requires regions to be patched specifically
+    Updating ExperienceConfig. Note that component cannot be updated once its created
     """
 
-    regions: List[PrivacyNoticeRegion]
+    regions: Optional[List[PrivacyNoticeRegion]] = Field(
```

Comment:
this now seems to be identical to what you've got in the `ExperienceConfigCreate` model, including the validator method. is there still a reason to have it split out here in the subclasses and not the base class? i see you've got a field `default` and `description` specified here that may not apply to the "create". still feels like there's some room to DRY this up a bit, perhaps?

Reply:
yes room for cleanup there! I'd originally treated regions differently for create and update in an earlier draft of this PR and later consolidated the behavior to reduce confusion.

Reply:
looks good, thanks for addressing!